app/views/foods/index.html.erb

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Recipe App</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", async: true %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+
+    <%= render "partials/nav" %>
+    
+    <section class="body">
+      <div class="add-btn">
+        <%= link_to 'foods/new', :remote => true do %>
+          <input type="button" class="btn btn-primary" value="Add Food">
+        <% end %>
+      </div>
+
```

Comment:
- [ ] When I click on the `Add food`, `Recipes`, and `General Shopping List` buttons before login in or registering a user I got the error. Please direct the user to the login page when a user clicks on those buttons before creating the account. 

![image](https://user-images.githubusercontent.com/55635977/157909439-21c96f42-0164-4ed2-9e16-9039c94e9e7a.png)

Reply:
resolved 👍